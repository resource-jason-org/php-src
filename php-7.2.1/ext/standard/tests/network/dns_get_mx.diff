003- Hosts: %i, weights: %i