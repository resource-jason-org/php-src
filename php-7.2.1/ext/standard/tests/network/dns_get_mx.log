
---- EXPECTED OUTPUT
Hosts: %i, weights: %i
Hosts: %i, weights: %i
Hosts: %i, weights: %i
---- ACTUAL OUTPUT
Hosts: 1, weights: 1
Hosts: 0, weights: 0
---- FAILED
