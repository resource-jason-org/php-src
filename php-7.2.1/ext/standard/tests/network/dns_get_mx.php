<?php
$domains = array('php.net', 'doc.php.net', 'wiki.php.net');
foreach ($domains as $domain) {
    if (getmxrr($domain, $hosts, $weights)) {
        echo "Hosts: " . count($hosts) . ", weights: " . count($weights) . "\n";
    }
}
?>
