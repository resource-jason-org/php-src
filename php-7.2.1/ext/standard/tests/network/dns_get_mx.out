Hosts: 1, weights: 1
Hosts: 0, weights: 0